core/src/main/java/org/restexpress/Request.java

```diff
@@ -19,12 +19,7 @@
 
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.ByteBufInputStream;
-import io.netty.handler.codec.http.FullHttpRequest;
-import io.netty.handler.codec.http.HttpHeaders;
-import io.netty.handler.codec.http.HttpMethod;
-import io.netty.handler.codec.http.HttpRequest;
-import io.netty.handler.codec.http.HttpVersion;
-import io.netty.handler.codec.http.QueryStringDecoder;
+import io.netty.handler.codec.http.*;
```

Comment:
Not everyone is fond of this minimalism (me included.) I think you should update this file (maybe others?) and keep the original author's way of importing classes.